internal/wspace/finder.go

```diff
@@ -19,23 +19,39 @@ import (
 	"strings"
 )
 
-const workspaceFile = "WORKSPACE"
+var workspaceFiles = []string{"WORKSPACE.bazel", "WORKSPACE"}
 
-// Find searches from the given dir and up for the WORKSPACE file
+// FindWorkspaceFile returns a path to a file in the provided root directory,
+// either to an existing WORKSPACE or WORKSPACE.bazel file, or to root/WORKSPACE
+// by default. Note that this function does NOT recursively check parent directories
+func FindWorkspaceFile(root string) string {
```

Comment:
Let's call this `FindWORKSPACEFile`. That should make it more clear that this is a `WORKSPACE` file, not a file in a workspace.